### DISCUSSION TITLE
Which physics model to use for mass4l and for other observable?

### DISCUSSION THREAD

**Discussion by ram1123** [vukasinmilosevic/Fiducial_XS#64] [Q&A]
Hi @tjavaid ,

There are two physics models present here: 

https://github.com/vukasinmilosevic/Fiducial_XS/tree/CMSSW_10_X/models

They are named as v2 and v3.

If I understand correctly, we need to run both models (v2 and v3) only for mass4l.

For all others( e.g. pT4l, leading jets, etc ), we just need to run v3. Right?

**Comment by tjavaid** [comment]
Hi @ram1123, for mass4l we should use both. `v2` would extract the cross sections in the channels and `v3` for total cross section in the inclusive bin `SigmaBin0`. 
`v3` is only used for differential observables where all `final states` are combine in a particular bin.
